spec/acceptance/self_service_spec.rb

```diff
@@ -77,12 +81,35 @@ class {'self_service':
         run_shell('rm -f /etc/puppetlabs/facter/facts.d/load_averages.json')
       end
 
-      it 'if S0007 conditions for false are met' do
+      it 'if S0007 and S0008 conditions for false are met' do
```

Comment:
`it` conditions should only check one thing. I'll rebase #7 after this is merged with an update since the code is already there.